packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -210,6 +213,54 @@ module.exports = (themeOptions = {}) => {
       'gatsby-plugin-remove-trailing-slashes',
       'gatsby-plugin-meta-redirect',
       'gatsby-plugin-netlify-cache',
+      {
+        resolve: 'gatsby-plugin-feed',
+        options: {
+          query: `
+          {
+            site {
+              siteMetadata {
+                title
+                description
+                siteUrl
+                site_url: siteUrl
+              }
+            }
+          }
+        `,
+          feeds: [
+            {
+              serialize: ({ query: { site, allReleaseNotePage } }) => {
+                return allReleaseNotePage.nodes.map((node) => {
+                  return {
+                    ...node,
+                    url: site.siteMetadata.siteUrl + node.slug,
+                    guid: site.siteMetadata.siteUrl + node.slug,
+                    custom_elements: [{ 'content:encoded': node.body }],
```

Comment:
MDX is not HTML, you can't just trow MDX into an RSS as HTML encoded content.   

I don't know if it's realistic to get a usable plaintext or "minimal HTML" representation directly from the GraphQL for our (sometimes custom) MDX, so I wold be fine omitting it as we currently do, too.  

But we can try - one option that publishers take is to put the excerpt until "read more" here so people move on to your actual website instead of staying in some feed consumer UI.  But even then it has to be either plaintext or plain semantic HTML